<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit_calc</class>
 <widget class="QDialog" name="Deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>441</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Сумма вклада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_dep_sum">
        <property name="minimum">
         <double>1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Срок (в мес.)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_dep_months">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Процентная ставка (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_dep_percent">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Налоговая ставка (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_dep_tax_rate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Периодичность выплат (в мес.)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_periodicity">
        <item>
         <property name="text">
          <string>Ежемесячно</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в квартал</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в полгода</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ежегодно</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_capitalization">
      <property name="text">
       <string>Капитализация</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Пополнения / снятия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Добавить +</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_depo">
      <column>
       <property name="text">
        <string>Номер месяца</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сумма пополнения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сумма снятия</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>21</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>481</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Сумма на вкладе к концу срока</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_sum_output">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Начисленные проценты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_percent_charges">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Сумма налога</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_tax_sum">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_error_dep">
      <property name="styleSheet">
       <string notr="true">color: rgb(200,66,71);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton_clear_dep">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_calc_dep">
        <property name="text">
         <string>Рассчитать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close_dep">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
